test/common/include/alpaka/test/KernelExecutionFixture.hpp

```diff
@@ -50,6 +51,14 @@ namespace alpaka
                             alpaka::workdiv::GridBlockExtentSubDivRestrictions::Unrestricted))
             {}
             //-----------------------------------------------------------------------------
+            KernelExecutionFixture(
+                WorkDiv const & workDiv) :
```

Comment:
Could we use inheriting constructors here and let the other one above inherit from this one?

Reply:
I originally tried that. But it turned out somewhat awkward to express in this case: the constructor above would need to not initialize members (as initialization in both constructors is not allowed), but it needs this values for the extent computation call.